app/controllers/identities_controller.rb

```diff
@@ -69,4 +80,29 @@ def show_sites # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity,
       render plain: "404 Not Found", status: :not_found
     end
   end
+
+  private
+
+  def flipper_params
+    params.require(:portal).to_sym
+  end
+
+  def form_user
+    identity = current_user.identity_scope.find_by(uid: params[:id])
+    identity.user.becomes(FormUser)
+  end
+
+  def admin?
+    current_user.user_actions&.dig("admin", "can_administer")
+  end
+
+  def switch_portal(portal, form_user)
```

Comment:
is this switching or toggling? something like toggle_feature might be a less confusing name

Reply:
agree, I created this switch only to get away from rubocop warnings and thought using `toggle` again would be confusing, will rewrite

Reply:
done, removing guard clause for _authorizing admin_ into before action / App controller helped with rubocop/method length